### PR TITLE
Bug: BAK-1091 CDN s3 url encoding fix

### PR DESCRIPTION
**Pull request by joel-wright**

### CODE REVIEW

**Review by PatFawbertMills — APPROVED** [review]
Makes sense.

File: backend/app/core/util.py [hunk 1]
```diff
@@ -14,6 +15,40 @@
     ".htm": "text/html",
     ".docx": "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
 }
+# Mappings sources from: https://github.com/GeorgePhillips/node-s3-url-encode/blob/master/index.js
```
[the remark this answers is not in the document]
Comment (joel-wright) [reply backend/app/core/util.py:18]: I don't think that's possible - `urlencode` is pretty opinionated & s3 URLs don't follow the spec :disappointed:

**Review by diversemix — APPROVED** [review]
lgtm :+1:

File: backend/app/core/util.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (diversemix) [reply backend/app/core/util.py:18]: dang!

[the remark this answers is not in the document]
Comment (joel-wright) [reply backend/app/core/util.py]: okay, have made another attempt by being very careful about what I url encode

**Comment by diversemix** [comment]
Looks good - I'll put some tests in here at some point ;)

**Pull request merged**